### PR TITLE
New layout of login page

### PR DESCRIPTION
**Pull request by ohdatboi**
Same as #215 but for login page.
login

### CODE REVIEW

**Comment by ohdatboi** [comment]
Fixed typos.
Screenshot:
loginfixed

File: public/styles/common.css [hunk 1]
```diff
@@ -351,17 +351,17 @@ ul.unmarkedList > li:hover,
 
 .light { opacity: 0.8; }
 
-#registerPackage {
+#inputForm {
```
Comment (MartinNowak) [inline public/styles/common.css]: Make it `.inputForm` then, no?

**Comment by MartinNowak** [comment]
We might want to evaluate using some simple CSS framework, such as http://getskeleton.com/, instead of writing everything from the ground up. No opinion on how important look-alike to dlang.org is though.

File: public/styles/common.css (hunk 1 above)
Comment (ohdatboi) [inline public/styles/common.css]: There are no cases with more than 1 input form on the page, but yes, making it `.inputForm` looks better. I'll fix it.

**Comment by ohdatboi** [comment]
@MartinNowak I was thinking about using CSS preprocessor (like SASS). This requires complete rewriting and rethinking styles, cleaning dead/unused rules, etc.
I would like to use some kind of framework, but framework *and* preprocessor are much better (avoiding copy/paste, adding some useful variables and more).

Anyway, there are not a lot of responses in #218, which adds resource builder. It should have more opinions, so bad/useless stuff won't get merged into master.

**Comment by wilzbach** [comment]
Currently:

image

New:

image

**Comment by wilzbach** [comment]
@s-ludwig 
1) Can we merge such trivial stuff without your approval?
2) Is there a trivial way to setup auto-deploy for the DUB registry?

**Comment by s-ludwig** [comment]
> Can we merge (or squash merge in this case) such trivial stuff without your approval?

No problem. I actually had this stored in my mind as merged because Martin had already approved it.

> Is there a trivial way to setup auto-deploy for the DUB registry?

We'd at least need to have a quick way to roll back changes in case of any regressions, which probably wouldn't qualify as trivial anymore. But I realized that I really need to invest some time into a better deployment and service management system anyway (better crash reports and redundancy to hide the effects of intermittent failures would also help a lot). I'll try to reserve some time for this next week.

**Comment by wilzbach** [comment]
> We'd at least need to have a quick way to roll back changes in case of any regressions, which probably wouldn't qualify as trivial anymore. 

Depending on the deployment time, it might be feasible to simply revert the PR and send it to the deployment queue.

> But I realized that I really need to invest some time into a better deployment and service management system anyway (better crash reports and redundancy to hide the effects of intermittent failures would also help a lot). I'll try to reserve some time for this next week.

If you use sth. like `ansible` and put your scripts here, I could probably hook the VPS, s.t. it's `staging.dub.pm`, i.e. we directly deploy all commits to master on it, but we only do releases whenever someone has time to look out in case of failures?

**Pull request merged**